gateway/admin.go

```diff
@@ -21,3 +31,148 @@ func (g *GatewayAdmin) Status() interface{} {
 	}
 
 }
+
+type GatewayRPCHandler struct {
+	jobsDB jobsdb.JobsDB
+}
+
+type SqlRunner struct {
+	dbHandle     *sql.DB
+	jobTableName string
+	err          error
```

Comment:
If we remove the err inside SqlRunner, this would make things compatible with what we already have.

Reply:
done